client/rechallenge/transformers.js

```diff
@@ -18,7 +19,13 @@ import {
 } from '../../common/utils/polyvinyl.js';
 import castToObservable from '../../common/app/utils/cast-to-observable.js';
 
-const babelOptions = { presets: [ presetEs2015, presetReact ] };
+/* NOTE: About stage-2 preset:
+ * Add stage-2 transform to enable es2016 class properties transform,
+ * i.e. to allow React class methods to be written as arrow functions.
+ * This is only needed in the React challenges, and since these are
+ * isolated in the buildChallengeEpic they could be fed with separate
+ * Babel presets. However, enabling everywhere is faster for now: */
+const babelOptions = { presets: [ presetEs2015, presetReact, stage2 ] };
```

Comment:
So this was unclear when we wrote the challenges which way we should go. The `class transforms` is for writing a class method with an arrow function to have the advantage of implicit `this` binding. Peter and I decided to add a challenge on it just to demonstrate the syntax (because we found it pretty common in other resources) but make it clear in the challenge that this is a future JavaScript version and the feature may not be added. We can remove it, but that was the motivation behind adding it originally.